### PR TITLE
Week1 rakshith

### PR DESCRIPTION
**Pull request by Rakshith356**

### CODE REVIEW

**Comment by Rakshith356** [comment]
ruby1-pic

**Comment by raosush** [comment]
@Rakshith356 we have updated out test, please pull the latest changes using `git pull`

File: week1/ruby-1.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+
+
+        </style>
+    </head>
+    <body>
+        <!-- Header -->
+        <header>
+            <figure>
+            <img src="C:\Users\admin\Documents\photo.jpg" alt="My Pic">
```
Comment (gaurang2001) [inline week1/ruby-1.html]: Unless the image is hosted elsewhere, always add relative paths to the image, so the code can work in any environment

**Comment by gaurang2001** [comment]
Please update your branch with changes from the `main` branch upstream using `git pull`, so the tests can pass. While you've completed your assignment, doing this will teach you how to work with `git` 😃

File: week1/ruby-1.html (hunk 1 above)
Comment (Rakshith356) [reply week1/ruby-1.html]: I have made the changes. Can u pls confirm?

**Review by gaurang2001 — APPROVED** [review]

**Comment by gaurang2001** [comment]
You've completed the Week 1 Assignment!!

**Pull request closed without merging**